src/app_charts/base/cloud/app-management.yaml

```diff
@@ -0,0 +1,210 @@
+{{ if eq .Values.app_management "true" }}
+# app-rollout-controller
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app-rollout-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: app-rollout-controller
+  template:
+    metadata:
+      labels:
+        app: app-rollout-controller
+    spec:
+      containers:
+      - name: app-rollout-controller
+        image: {{ .Values.registry }}{{ .Values.images.app_rollout_controller }}
+        args:
+        - "--params=\
+          domain={{ .Values.domain }},\
+          project={{ .Values.project }},\
+          ingress_ip={{ .Values.ingress_ip }},\
+          registry={{ .Values.registry }},\
+          deploy_environment={{ .Values.deploy_environment }},\
+          certificate_authority.crt={{ .Values.certificate_authority.crt }}"
+        - "--webhook-port=9876"
+        - "--cert-dir=/tls"
+        env:
+        - name: GOOGLE_CLOUD_PROJECT
+          value: {{ .Values.project }}
+        ports:
+        - name: webhook
+          containerPort: 9876
+        volumeMounts:
+        - mountPath: /home
+          name: home
+        - mountPath: /tls
+          name: tls
+      initContainers:
```

Comment:
we probably only need this for the chart assignment controller. The approll-out is only creating the chart assignments.

Reply:
Oh yes, that's true. We could remove the helm binary from app-rollout-controller too. I'll test that.

Reply:
This is working. I removed the helm binary, initContainer and /home volume mount from app-rollout-controller.